packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardDashboard/TransactionsBox/TransactionsBox.template.tsx

```diff
@@ -119,7 +118,11 @@ const TransactionsBox = ({ modalActions }) => {
   }
 
   const generateTransactionIcon = (type) => {
-    return type === TransactionType.PAYMENT ? <IconMinusCircle /> : <IconPlusCircle />
+    return type === TransactionType.PAYMENT ? (
+      <IconMinusCircle color={PaletteColors['blue-300']} label='Spent' size='small' />
+    ) : (
+      <IconPlusCircle color={PaletteColors['blue-300']} label='Spent' size='small' />
+    )
```

Comment:
Same as commented some place above, what about reducing the amount of ternary operators clogging everything?

```jsx
const generateTransactionIcon = (type) => {
  const IconComponent = type === TransactionType.PAYMENT ? IconMinusCircle : IconPlusCircle;
  return <IconComponent color={PaletteColors['blue-300']} label='Spent' size='small' />
}
```